doc/admin_doc/install_doc/installation.rst

```diff
@@ -77,30 +77,88 @@ You can now start and enable the ``mongod`` and ``docker`` services:
+
+    sudo apt install curl gnupg lsb-release
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+    echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    sudo apt update
+    sudo apt install docker-ce docker-ce-cli
+
+Then, Mongo:
 
 .. code-block:: bash
 
-    # apt-get update
-    # apt-get install git gcc tidy python3 python3-pip python3-dev libzmq3-dev mongodb apt-transport-https curl gnupg lsb-release
-    # curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
-    # echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
-    # apt-get update
-    # apt-get install docker-ce docker-ce-cli
+    wget -qO - https://www.mongodb.org/static/pgp/server-6.0.asc | sudo apt-key add -
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list
+    sudo apt update
+    wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+    sudo dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+    sudo apt install -y mongodb-org
+
+.. NOTE::
 
-You may also add ``libxmlsec1-dev libltdl-dev`` for the SAML2 auth plugin
+    Libssl installation is a temporary fix that is not required for all versions.
+    It may not work anymore (or may not be necessary) for future versions.
+
+You may also add ``libxmlsec1-dev libltdl-dev`` for the SAML2 auth plugin.
 
 You can now start and enable the ``mongod`` and ``docker`` services:
 ::
 
-    # systemctl start mongodb
-    # systemctl enable mongodb
+    # systemctl daemon-reload
+    # systemctl start mongod
+    # systemctl enable mongod
     # systemctl start docker
     # systemctl enable docker
 
+And make docker available for non-root user:
+
+1. Run the groupadd command below to create a new group called docker. Enter your password to continue running the command.
+
+.. code-block:: bash
+
+    sudo groupadd docker
+
+If the docker group exists in the user group, you will get a message like "group already exists".
+
+2. Next, run the usermod command below where the -aG options tell the command to add your user account (user) to the (docker) group.
+This command causes your user account to have non-user access.
+
+.. code-block:: bash
+
+    sudo usermod -aG docker user
+
+3. Run the newgrp command below to change the current real group ID to the docker group.
+
+.. code-block:: bash
+
+    sudo newgrp docker
+
+If, at this point, you’re still getting a permission error, then consider giving more access to the docker.sock file.
```

Comment:
I would not recommend that, this can be a severe security issue in shared environments.